subworkflows/local/utils_nfcore_eager_pipeline/main.nf

```diff
@@ -81,55 +81,82 @@ workflow PIPELINE_INITIALISATION {
     //
     // Create channel from input file provided through params.input
     //
-    ch_samplesheet_for_branch = Channel.fromSamplesheet("input")
-                                    .map{
+    ch_samplesheet = Channel.fromSamplesheet("input")
+                                    .map {
                                         meta, r1, r2, bam ->
                                             meta.single_end = meta.pairment == "single" ? true : false
+                                            meta.id = meta.sample_id
```

Comment:
Should be new mapping rather than overwriting, e.g.

```
meta + [id: meta.sample_id]
```

or similar

Reply:
I'm super paranoid about this now, sorry 😅

Reply:
This adds a new meta field and does not overwrite.